data-prepper-plugins/peer-forwarder/src/main/java/com/amazon/dataprepper/plugins/processor/peerforwarder/PeerForwarderConfig.java

```diff
@@ -37,6 +43,20 @@ public static PeerForwarderConfig buildConfig(final PluginSetting pluginSetting)
         final HashRing hashRing = new HashRing(peerListProvider, NUM_VIRTUAL_NODES);
         final PeerClientPool peerClientPool = PeerClientPool.getInstance();
         peerClientPool.setClientTimeoutSeconds(3);
+        final boolean ssl = pluginSetting.getBooleanOrDefault(SSL, DEFAULT_SSL);
+        final String sslKeyCertChainFilePath = pluginSetting.getStringOrDefault(SSL_KEY_CERT_FILE, null);
+        final File sslKeyCertChainFile;
+        if (ssl) {
+            if (sslKeyCertChainFilePath == null || sslKeyCertChainFilePath.isEmpty()) {
+                throw new IllegalArgumentException(String.format("%s is enable, %s can not be empty or null", SSL, SSL_KEY_CERT_FILE));
+            } else {
+                sslKeyCertChainFile = new File(sslKeyCertChainFilePath);
```

Comment:
Nit: May want to add another OR branch in the above if statement to handle cases where the File cannot be found.

Reply:
Sure.